templates/firstboot/mellanox_fw_update.yaml

```diff
@@ -13,6 +13,13 @@ parameters:
     type: boolean
     default: False
     description: "Force update the fw even if it's older version"
+  DEV_WHITE_LIST:
+    type: list
+    default: []
+    description: list of MLNX devices PCIs to be processed.
+                 If the value is empty, all MLNX devices will be processed.
+                 Example, ['0000:04:00.0', '0000:81:00.0']
+                 Make sure to choose only the PCI ends with 0
```

Comment:
what about applying per PF configurations ? the code no only updates firmware, 
you should allow wildcard for the function.

we assume SR-IOV is not configured when this bit runs right ?

Reply:
Bt fw is per nic? So why this a problem?

Reply:
Bt fw is per nic? So why this a problem?

Reply:
nvconfig may be per PF

Reply:
Yes but fw is per nic, so I would to keep it all config per nic, not per pf

Reply:
the issue i described is not related to this commit, nvm.
before the change we only processed PF 0 as well

Reply:
right, there is not requirement for that at least for now :)